src/sentry/web/frontend/debug/debug_performance_issue.py

```diff
@@ -50,6 +50,7 @@ def get(self, request):
             perf_event = perf_event_manager.save(project.id)
 
         perf_event = perf_event.for_group(perf_event.groups[0])
+        perf_event.group.id = 1  # hard code to 1 for acceptance tests
```

Comment:
@ceorourke why not hard code it to `2` since that's what it's supposed to be?

Reply:
Would it make a difference? The fixture just needs to match the group ID that's passed.

Reply:
`2` was likely what happened to work at the time, it's not particularly meaningful otherwise. `1` is what we have hardcoded for error alerts too

Reply:
@ceorourke just makes it easier to remove the hack later was my thinking